app/styles/new-signup.css

```diff
@@ -0,0 +1,88 @@
+.container {
+    width: 100%;
+    color: #666;
+    text-align: center;
+    padding: 1em 0;
+}
+
+.user-details {
+    display: flex;
+    padding: 2rem;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    max-width: 80%;
+    margin: auto;
+    background-color: #f0f2f5;
+}
+
+.user-details>* {
+    padding: 1rem;
+}
+
+.user-details__heading {
+    font-size: 2rem;
+    color: black;
+    margin-bottom: 0;
+}
+
+.user-details__field input {
+    padding: 0.8rem;
+    width: 100%;
+    border-radius: 10px;
+    border: 2px solid darkgray;
+    font-size: 2rem;
+}
+
+.signup-action__btn button {
+    padding: .75rem 2rem;
+    border-radius: 10px;
+    outline: none;
+    background-color: green;
+    color: white;
+    text-decoration: none;
+    cursor: pointer;
+    width: 40%;
+    display: block;
+    margin: auto;
+    font-size: 2rem;
+    border: none;
+}
+.error {
+    color: red;
+    padding: 0.4rem;
+    font-size: 1rem;
+}
+.signup-action__btn button.btn-disabled {
+    background: #ccc;
+}
+
+@media (max-width: 500px) {
+    .user-details__heading {
+        font-size: 1.4rem;
+    }
+    .user-details__field input {
+        font-size: 1.5rem;
+    }
+    .signup-action__btn button {
+        font-size: 1.5rem;
+    }
+}
+
+@media (max-width: 768px) {
+    .user-details {
+        width: 80%;
+    }
+}
+
+@media (max-width: 568px) {
+    .user-details {
+        width: 100%;
+    }
+}
+
+@media (max-width: 520px) {
+    .signup-action__btn button {
+        width: 70%;
+    }
+}
```

Comment:
Can we please fix these media queries and have better breakpoints ?
For reference
![image](https://user-images.githubusercontent.com/78433013/190211601-e2861fe8-ff1f-4a6f-8032-e84ad0d79b79.png)

Reply:
thanks will do that